compose/src/main/java/com/bottlerocketstudios/compose/home/HomeCard.kt

```diff
@@ -88,6 +98,62 @@ fun CardLayout(userRepositoryUiModel: UserRepositoryUiModel, selectItem: (userRe
     }
 }
 
+@Composable
+fun HomePullRequestItemCard(state: PullRequestItemState) {
```

Comment:
@LuisRocketRangel NIT: super minor and it looks like it might be a little tricky to reuse CardLayout above this for this PR card layout but maybe just to make things a little easier to read we can rename CardLayout to RepositoryCardLayout and then rename HomePullRequestItemCard to PullRequestCardLayout.

Reply:
Will fix!